Updated troubleshooting section

DanGLVK opened the pull request:
Still needs slight revisions due to lack of proper Reddit guide links

DanGLVK committed:
Updated troubleshooting section 

Still needs slight revisions due to lack of proper Reddit guide links

KobeW50 commented on general-guides/community-wiki/yt-troubleshooting.md:
```diff
@@ -1,34 +1,27 @@
 ### **1. No internet connection / The home page isn't loading**
 
-Navigate to your device settings > Accounts > Manage accounts > Vanced MicroG > Remove account. You can sign in again if you want.
+Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.
```
On my device (Samsung, running A14, OneUI 6) my GmsCore account appears similar to my Google account, and it does not say "GmsCore"

Only the logo is different.

![Screenshot_20240521_141952_Settings](https://github.com/ReVanced-Extended-Community/Community-Guides/assets/84587632/932a0514-3efa-4d6c-a7e1-e01333b54e4e)

I don't see a simple way of explaining this to users without showing a screenshot of what the logo is logo is. LMK if I should create and upload an image (I've been doing it through imgur for some reason 🤷)

DanGLVK replied:
Done

replaced it with Google (blue icon), is this good enough?

KobeW50 commented on general-guides/community-wiki/yt-troubleshooting.md:
```diff
@@ -1,34 +1,27 @@
+Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.
 
-> If that doesn't work you can try to uninstall Vanced MicroG and reinstall [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases/latest) or [inotia00's Vanced MicroG fork](https://github.com/inotia00/VancedMicroG/releases/latest).
+> If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) installed.
```
The purpose of uninstalling is to forcefully remove the user's account from the device. So it should be:
 
```suggestion
> If that doesn't work you can uninstall GmsCore and then reinstall [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).

```

KobeW50 commented on general-guides/community-wiki/yt-troubleshooting.md:
```diff
@@ -1,34 +1,27 @@
 ### **3. The download feature isn't working**
 
-Refer to step 4 of the [YT guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/yt-guide.md#downloader-setup) to understand and set up the download function in YT ReVanced Extended.
+Refer to step 4 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2b-downloader-setup) to understand and set up the download function in YT ReVanced Extended.
```
When linking to a part of a good it is good practice to link to a general section (ie: section 2, instead of 2b) because the link may break if the page is changed.

```suggestion
Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YT ReVanced Extended.

```

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -2,179 +2,40 @@
-### **1. The download feature isn't working**
-
-Refer to step 4 of the [YTM guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/ytm-guide.md#downloader-setup) to understand and set up the download function in YTM Extended.
+> If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) installed.
```
```suggestion
> If that doesn't work you can uninstall GmsCore and then reinstall [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).

```

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -2,179 +2,40 @@
+### **3. The download feature isn't working**
 
-**b)** Disable any VPNs or ad-blockers and restart the RVX Manager.
+Refer to step 2b of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/wiki-redesign/general-guides/community-wiki/patching%20%26%20setup%20info.md#2b-downloader-setup) to understand and set up the download function in YTM Extended.
```
```suggestion
Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YTM Extended.

```

KobeW50 commented on general-guides/community-wiki/yt-troubleshooting.md:
```diff
@@ -1,34 +1,27 @@
+Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.
 
-> If that doesn't work you can try to uninstall Vanced MicroG and reinstall [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases/latest) or [inotia00's Vanced MicroG fork](https://github.com/inotia00/VancedMicroG/releases/latest).
+> If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) i
```
I think we can combine this issue with `#1`. So it can just say something like:

 ```
### **2. I'm having issues signing in to my Google account**

Follow the steps in issue **#1** above.
```

What do you think? (Same thing for the ytm troubleshooting)

DanGLVK replied:
Done, check if it's good enough

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -2,179 +2,40 @@
-Continue from the middle of step 2 in the [YT guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/ytm-guide.md#2-patching-the-apk).
+Navigate to your device settings > Accounts > Manage accounts > GmsCore > Remove account. You can sign in again if you want.
 
+> If that doesn't work make sure you have the latest version of [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest) i
```
Refer to my comment on issue 2 in the yt troubleshooting page.

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -2,179 +2,40 @@
+### **4. How do I get YTM Extended on Android Auto?**
 
-**a)** Make sure you are using the latest [RVX Manager](https://github.com/inotia00/revanced-manager/releases/latest).
+Refer to step 3a of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3aandroid-auto-setup).
```
```suggestion
Refer to section 3 of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3-yt-music-setup).

```

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -2,179 +2,40 @@
+Google has made it so that Canadian users who do not have YT Premium can only use the Radio feature within YouTube Music.
 
-**c)** The email you are signing in with has parental control restrictions.
+Refer to step 3b of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3b-spoof-app-version-setup-for-canadian-users) to solve this.
```
```suggestion
Refer to section 3 of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3-yt-music-setup) to solve this.

```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+You'll need to patch on a different Android device or on PC.
+
+When patching on a different device or PC, be sure to patch the APK variant that was built for the architecture of the device you plan to install the patched APK on. For YouTube, you can simply use the universal version, but for YouTube Music, you'll need the specific version corresponding to your device.
```
```suggestion
YT Music:
> Note that when patching YT Music on a different device or PC, you'll need to patch the APK variant that was built for the architecture of the device you plan to **install** the patched APK on. 
```

I haven't yet added the info for selecting the right APK to the setup info guide, but after I complete it this can link to it.

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+**3)** Open the RVX Manager > Patcher > Select an application > **YouTube/YT Music**.
+
+Continue from the middle of step 2 in the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners/).
```
This entire section should ideally also be written to include Reddit, but it makes it complicated (since the Reddit package name isn't changed when it is patched).

LMK what you think about this:

```suggestion
### **3. Nothing happens when I select the APK from storage**

Make sure you are using the latest [RVX Manager](https://github.com/inotia00/revanced-manager/releases/latest). If you already are using the correct version, continue with the following steps:

**1)** Uninstall the official app you are trying to patch. 

> If you cannot uninstall it, go to the App info of the app > Options menu (as shown in [this image](https://imgur.com/a/0js3AZR)) > Uninstall updates.

**2)** Install the APK that you downloaded. (Now you should have the suggested APK installed.)

**3)** Open the RVX Manager > Patcher > Select an application, and select the app from the app list.

Continue from the middle of step 2 in the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners/).
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+**a)** Make sure you are using the latest [RVX Manager](https://github.com/inotia00/revanced-manager/releases/latest).
+
+**b)** Make sure you are using the suggested Youtube/YT Music version given in step 1 of the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners/).
```
```suggestion
**b)** Make sure you are using the suggested APK version given in step 1 of the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners/).
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+> If you patch with the RVX Manager on a different device you'll need to export the patched APK so that you can send it to the device you plan to install it on. To export the APK, tap the save icon when patching is completed as shown in [this image](https://imgur.com/a/FKD0okE). Then press "Save".
+
+If you want to patch on PC, follow the instructions for the desired method in in [inotia00's documentation](https://github.com/inotia00/revanced-documentation#revanced-extended-documentation).
```
```suggestion
* If you want to patch on a different device refer to the info in issue **#1** above.
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+### **6. Manager is stuck on "Installing..."**
+
+Export the patched APK by tapping the save icon as shown in [this image](https://imgur.com/a/FKD0okE). Save it, and then install the exported APK from your file manager. You might get an error while installing it, if so refer to the entry below for instructions on how to solve the issue.
```
This bug doesn't occur anymore

```suggestion

```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+   - If you see a Google Play Protect pop-up, tap **"More details"** and then **"Install anyway"**. Do not tap **"Got it"**, as this will cancel the installation. [View Image](https://imgur.com/a/Ck8nfhn).
+
+2. **Ensure Sufficient Storage:**
+    - Make sure your device has enough storage space
```
```suggestion
2. **Ensure Sufficient Storage:**
    - Make sure your device has enough storage space.
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+
+
+### **7. "App not installed."**
```
This issue should ideally also have Reddit in mind.

For parts of the checklist specific to YT/YT Music, you can put it as a quoted note thing (idk the term lol). Look at section 2b in the setup info guide to see how I divided the info specific to yt and ytm

https://github.com/ReVanced-Extended-Community/Community-Guides/blob/wiki-redesign/general-guides/community-wiki/patching%20%26%20setup%20info.md#2b-downloader-setup

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+**b)** There was a failure or error while it was patching and you installed the broken APK. Refer to troubleshooting issue **#6** if this occurred.
+
+**c)** The email you are signing in with has parental control restrictions.
```
The email one is specific to YTM.

DanGLVK replied:
Done

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,114 @@
+
+If the app is crashing on startup, there are 3 common causes of it:
+
+**a)** You do not have GmsCore installed. Download and install [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).
```
Ofc this is specific to YT/YT Music

```suggestion
YouTube and YT Music:
> **a)** You do not have GmsCore installed. Download and install [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).
```

KobeW50 commented on general-guides/community-wiki/troubleshooting-index.md:
```diff
@@ -1,5 +1,7 @@
\ No newline at end of file
+[My issue is related to an **YT ReVanced Extended** specific issue.](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/yt-troubleshooting.md#yt-revanced-extended-troubleshooting)
+
+[My issue is related to an **YT Music ReVanced Extended** specific issue.](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/ytm-troubleshooting.md#yt-music-extended-troubleshooting)
```
```suggestion
[My issue is related to a **YT ReVanced Extended** specific issue.](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/yt-troubleshooting.md#yt-revanced-extended-troubleshooting)

[My issue is related to a **YT Music ReVanced Extended** specific issue.](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/ytm-troubleshooting.md#yt-music-extended-troubleshooting)

```

DanGLVK committed:
Adjust text to clarify about the need for uninstall

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Link to general section

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Link to general section (2)

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Add missing punctuation

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Correct account name

GmsCore uses the old blue Google icon to distinguish itself from the main Google account

DanGLVK committed:
Correct account name (2)

GmsCore uses the old blue Google icon to distinguish itself from the main Google account

DanGLVK committed:
Merge troubleshooting steps related to account login/GmsCore (1)

DanGLVK committed:
Merge troubleshooting steps related to account login/GmsCore (2)

DanGLVK committed:
Link to general section (3)

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Clarify GmsCore steps as specific to Youtube/YT Music

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
typo fix

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Email section is an YT-Music specific issue

DanGLVK committed:
Remove issue that's YT-Music specific

DanGLVK committed:
Update general-guides/community-wiki/general-troubleshooting.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Clarify GmsCore uninstall

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Generalize storage APK section

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Generalize patch aborting troubleshooting

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Update general-guides/community-wiki/general-troubleshooting.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Remove outdated issue

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Fix order after issue removal

DanGLVK committed:
Separate Youtube/YT-Music specific install troubleshooting

DanGLVK committed:
Generalize install issue

DanGLVK committed:
Update general-troubleshooting.md

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,112 @@
+
+> If you cannot uninstall it, go to the App info of the app > Options menu (as shown in [this image](https://imgur.com/a/0js3AZR)) > Uninstall updates.
+
+**2)** Install the APK that you downloaded. (Now you should have the suggested APK installed.)
```
My fault

```suggestion
**2)** Install the APK that you downloaded from APKMirror.
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,112 @@
+
+Make sure you are using the latest [RVX Manager](https://github.com/inotia00/revanced-manager/releases/latest). If you already are using the correct version, continue with the following steps:
+
+**1)** Uninstall the official app you are trying to patch. 
```
My fault

```suggestion
**1)** Uninstall the unpatched YouTube / YT Music / Reddit from your device. 
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,112 @@
+
+**a)** Make sure you are using the latest [RVX Manager](https://github.com/inotia00/revanced-manager/releases/latest).
+
+**b)** Disable any VPNs or ad-blockers and restart the RVX Manager.
```
```suggestion
**b)** Disable any ad-blockers or VPNs and restart the RVX Manager.
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,112 @@
+
+**c)** If your device's language was set to a language that reads right-to-left when the failure occurred then set your device's language to English or any language that reads left-to-right. Then fully close the Manager, and try again.
+
+* If it still doesn't work you can try searching and asking on the [subreddit](https://www.reddit.com/r/revancedextended/) or [Telegram group](https://t.me/revanced_extended_chat) for your error. Alternatively, you can patch on a different device or use a different patching method.
```
```suggestion
* If it still doesn't work you can try searching and asking for support in the [subreddit](https://www.reddit.com/r/revancedextended/) or [Telegram group](https://t.me/revanced_extended_chat). Alternatively, you can patch on a different device or use a different patching method.
```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,112 @@
+
+
+
+### **6. "App not installed."**
+
+If you encounter any sort of "App not installed" error during the installation of the patched app, follow these steps to resolve the issue:
+
+1. **Google Play Protect Warning:**
+   - If you see a Google Play Protect pop-up, tap **"More details"** and then **"Install anyway"**. Do not tap **"Got it"**, as this will cancel the installation. [View Image](https://imgur.com/a/Ck8nfhn).
+
+2. **Ensure Sufficient Storage:**
+    - Make sure your device has enough storage space.
+
+3. **Uninstall Existing Versions:**
+   - Remove any existing patched apps, including those in alternate user profiles or private folders.
+
+ > Youtube/YT Music
+> 
+>  - If you are trying to install the unpatched Youtube/YT Music APK from APKMirror.com, note that it is not necessary to install it directly. Instead, follow step 2 in the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners) to select and patch the APK correctly.
+
+
+4. **Check APK Compatibility:**
+   - Ensure that the patched APK matches the architecture of your device. Refer to step 1 in the [general guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/ytm-guide.md#1-downloading-rvx-manager-yt-music-apk--vanced-microg) for information on downloading the correct APK.
+
+> Youtube/YT Music
+>  - If you are trying to install the unpatched Youtube/YT Music APK from APKMirror.com, note that it is not necessary to install it directly. Instead, follow step 2 in the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners) to select and patch the APK correctly.
+
+**Youtube/YT Music specific issues**
+
+5. **Patch with `GmsCore Support`:**
+   - Ensure that the `GmsCore Support` patch is applied when patching the APK.
+
+6. **GmsCore Version:**
+   - Ensure you are not installing an older version of GmsCore than the one already installed. Uninstall the current version via Settings > Apps > GmsCore > Uninstall.
+   - Make sure to download the latest GmsCore from [here](https://github.com/ReVanced/GmsCore/releases/latest)
+
+By following these steps, you should be able to resolve the majority of installation issues with Youtube/YT Music Extended.
```
```suggestion
### **6. Installation errors**

If you encounter any installation error when installing the patched app, follow these steps until the issue is resolved:

1. **Google Play Protect Warning:**
If you see a Google Play Protect pop-up, tap **"More details"** and then **"Install anyway"**. Do not tap "Got it", as this will cancel the installation. [View Image](https://imgur.com/a/Ck8nfhn).

2. **Ensure Sufficient Storage:**
Make sure your device has enough storage space.

3. **Uninstall Existing Versions:**
Remove any existing installation of the app you are trying to install, even if it is in an alternate user profile or private folder.

4. **Check The APK**
If you are trying to install the unpatched APK from APKMirror.com, note that it is simpler to avoid installing it before patching. Instead, follow step 2 in the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners) to select and patch the APK without installing it beforehand.

Youtube / YT Music:
> 5. **Patch with `GmsCore Support`:**
> Ensure that the `GmsCore Support` patch is applied when patching the APK.

YT Music:
> 6. **Check APK Compatibility:**
> Ensure that the APK you patched is built for your device's architecture. Refer to TODO for information on downloading the correct APK.

```

KobeW50 commented on general-guides/community-wiki/general-troubleshooting.md:
```diff
@@ -0,0 +1,112 @@
+
+
+
+### **7. App crashes on startup**
+
+If the app is crashing on startup, there are 3 common causes of it:
+
+YouTube and YT Music:
+> **a)** You do not have GmsCore installed. Download and install [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).
+
+**b)** There was a failure or error while it was patching and you installed the broken APK. Refer to troubleshooting issue **#6** if this occurred.
+
```
```suggestion
### **7. App crashes on startup**

If the app is crashing on startup, it may be because of one of the following:

1. There was a failure or error while it was patching and you installed the broken APK. Refer to troubleshooting issue **#5** if this occurred.

YouTube / YT Music:
> 2. You do not have GmsCore installed. Download and install [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).
>
> 3. The `GmsCore support` patch was not applied to the app.

YT Music:
> 4. The email you are signing in with has parental control restrictions.

```

KobeW50 commented on general-guides/community-wiki/troubleshooting-index.md:
```diff
@@ -1,5 +1,8 @@
 **What are you troubleshooting?**
 
-[My issue is related to **YT ReVanced Extended** or its installation process.](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/yt-troubleshooting.md#yt-revanced-extended-troubleshooting)
+[My issue is related to the installation process.](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/general-troubleshooting.md#General-Troubleshooting)
```
```suggestion
[My issue is related to the patching or installation processes.](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/general-troubleshooting.md#General-Troubleshooting)

```

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -2,33 +2,28 @@
 
+### **2. The download feature isn't working**
 
+Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YTM Extended.
```
```suggestion
Refer to section 2 of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#2-general-app-and-feature-setup) to understand and set up the download function in YT Music RVX.

```

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -2,33 +2,28 @@
-> If that doesn't work you can try to uninstall Vanced MicroG and reinstall [Vanced MicroG](https://github.com/TeamVanced/VancedMicroG/releases/latest) or [inotia00's Vanced MicroG fork](https://github.com/inotia00/VancedMicroG/releases/latest).
 
 
+### **3. How do I get YTM Extended on Android Auto?**
```
```suggestion
### **3. How do I get YT Music RVX on Android Auto?**

```

DanGLVK replied:
Is the app name really YT Music RVX? Bit convoluted huh?

KobeW50 commented on general-guides/community-wiki/ytm-troubleshooting.md:
```diff
@@ -37,144 +32,11 @@ Navigate to your device settings > Accounts > Manage accounts > Vanced MicroG >
-
-
-
+Refer to step 3b of the [setup info guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3b-spoof-app-version-setup-for-canadian-users) to solve this.
```
```suggestion
Refer to section 3 of the [setup guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/patching%20%26%20setup%20info.md#3-yt-music-setup) to solve this.

```

KobeW50 committed:
Clarify which APK to install when troubleshooting

KobeW50 committed:
Clarify which APK to uninstall when troubleshooting

KobeW50 committed:
Update general-guides/community-wiki/general-troubleshooting.md

DanGLVK committed:
Update general-guides/community-wiki/general-troubleshooting.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Update general-guides/community-wiki/general-troubleshooting.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Update general-guides/community-wiki/ytm-troubleshooting.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

KobeW50 committed:
Update general-guides/community-wiki/ytm-troubleshooting.md

KobeW50 committed:
Merge troubleshooting steps related to account login/GmsCore (3)d

DanGLVK committed:
Update general-guides/community-wiki/troubleshooting-index.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Update general-guides/community-wiki/general-troubleshooting.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

DanGLVK committed:
Update general-guides/community-wiki/ytm-troubleshooting.md

Co-authored-by: KobeW50 <84587632+KobeW50@users.noreply.github.com>

KobeW50 commented:
> Still needs slight revisions due to lack of proper Reddit guide links

What do you mean?

DanGLVK commented:
> > Still needs slight revisions due to lack of proper Reddit guide links
> 
> What do you mean?

Was already addressed in your reviews, so ignore it

KobeW50 commented:
Thank you. I will continue to work on this branch a bit more and then merge to main sometime this week.

DanGLVK commented:
> Thank you. I will continue to work on this branch a bit more and then merge to main sometime this week.

Tbh i feel like you did most of the work here 😅

KobeW50 commented:
> Tbh i feel like you did most of the work here 😅

Absolutely not. Literally all you

I really appreciate it
